gunicorn/config.test.py

```diff
@@ -143,11 +148,20 @@
 #       A string of "debug", "info", "warning", "error", "critical"
 #
 
+
+class CustomLogger(Logger):
+    def now(self):
+        return datetime.datetime.now(
+            tz=pytz.timezone("Europe/London")
+        ).strftime("%d-%b-%y %H:%M:%S")
+
+
+logger_class = CustomLogger
+
 errorlog = "-"
 loglevel = "info"
 accesslog = "-"
-# access_log_format = '%(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"' # noqa
-access_log_format = "{'remote_ip':'%(h)s','request_id':'%({X-Request-Id}i)s','response_code':'%(s)s','request_method':'%(m)s','request_path':'%(U)s','request_querystring':'%(q)s','request_timetaken':'%(D)s','response_length':'%(B)s'}"  # noqa
+access_log_format = '{"logType": "gunicorn-access", "time": "%(t)s", "remote_ip":"%(h)s","request_id":"%({X-Request-Id}i)s","response_code":"%(s)s","request_method":"%(m)s","request_path":"%(U)s","request_querystring":"%(q)s","request_timetaken":"%(D)s","response_length":"%(B)s"}'  # noqa
```

Comment:
I don't think we need `time` here. PaaS will provide us with a timestamp from the syslog. See https://docs.cloud.service.gov.uk/monitoring_apps.html#configure-logstash-filters for reference.

Reply:
Is knowing time (as the application environment has it set) ever useful to confirm any tz config errors?

Reply:
I think this might confuse things as PaaS timestamp will be UTC and this is London time. Can we remove it, and then we can add it back if we find a use case in future?

Reply:
Great, sure, removed